WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -28,6 +28,7 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
 
     @Published private(set) var isAddMoreDisabled: Bool = false
     @Published var isExitPOSDisabled: Bool = false
+    @Published var areTotalsFullScreen: Bool = false
```

Comment:
I am having a hard time naming this property.
Basically this boolean is used to determine if the whole totals/payments view is full screen (cart is not showed).
Anybody has any idea for a better name or is this ok?

Reply:
How about something like `isTotalsViewFullScreen`? A comment about the variable like what you descirbed would be helpful too.